packages/terra-base/package.json

```diff
@@ -28,8 +28,7 @@
     "react-dom": "^15.4.2"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "react": "^15.4.2"
```

Comment:
I am removing `react` as a dependency for several of our packages. I am noticing this is forcing multiple nested versions of `react` to be installed.